cmd/crio/main.go

```diff
@@ -197,9 +197,9 @@ func main() {
 			Name:  "conmon",
 			Usage: "path to the conmon executable",
 		},
-		cli.BoolFlag{
-			Name:  "debug",
-			Usage: "enable debug output for logging",
+		cli.StringFlag{
+			Name:  "log-level",
+			Usage: "log CRI-O messages above specified level: debug, warn, error, fatal or panic",
```

Comment:
sorry @sameo, shoud have spotted this earlier.  Is the level set "above" the specified level or "at" the specified level?  Should "above" be changed to "at the"?

Reply:
or perhaps "to the"

Reply:
It really is inclusive. For example setting log-level to warn will include all warn, error, fatal and panic messages.

Reply:
Right and in my mind if you say "above" here, then when I set it to warn, I wouldn't expect to get get warning, but the levels above that , error, fatal and panic.  I'd vote "to the", but I'm just one vote.